authors/apps/profiles/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from .views import ProfileView
+
+
+app_name = "profiles"
+urlpatterns = [
+    path('profiles/<str:username>/', ProfileView.as_view(), name='put-profile'),
```

Comment:
What's wrong with using <username> without the str?

Reply:
Please clarify.

Reply:
WHy not just <username> instead of <str:username>